pages/index.js

```diff
@@ -0,0 +1,395 @@
+                    fast subscription pages
+                  </a>
+                </li>
+                <li>
+                  Accept a one-time payment on{" "}
+                  <a href="https://github.com/stripe-samples/accept-a-payment">
+                    web, iOS, or Android
+                  </a>
+                  .
+                </li>
+              </ul>
+            </div>
+            <a
+              href="https://github.com/stripe-samples"
+              className="button"
+              target="_blank"
+              rel="noreferrer"
+            >
+              Explore samples
+            </a>
+          </div>
+        </section>
+
+        <h2>Developer resources</h2>
+        <section className="tools">
+          <a
+            href="https://stripe.com/docs/development/quickstart"
+            target="_blank"
+            rel="noreferrer"
+          >
+            <h3 className="uppercase">Development quickstart</h3>
+            <p>
+              Obtain API keys, install a client library, and make a test API
+              request.
+            </p>
+          </a>
+          <a
+            href="https://stripe.com/docs/payments/integration-builder"
+            target="_blank"
+            rel="noreferrer"
+          >
+            <h3 className="uppercase">Integration builder</h3>
+            <p>
+              Learn how to embed a custom Stripe payment form in your website or
+              application.
+            </p>
+          </a>
+          <a
+            href="https://stripe.com/docs/api"
+            target="_blank"
+            rel="noreferrer"
+          >
+            <h3 className="uppercase">API reference</h3>
+            <p>
+              View all backend API objects, methods, attributes, and responses.
+            </p>
+          </a>
+          <a
+            href="https://checkout.stripe.dev/"
+            target="_blank"
+            rel="noreferrer"
+          >
+            <h3 className="uppercase">Checkout demo</h3>
+            <p>
+              Choose your use case for a secure, Stripe-hosted payment page.
+            </p>
+          </a>
+          <a
+            href="https://www.youtube.com/stripedevelopers"
+            target="_blank"
+            rel="noreferrer"
+          >
+            <h3 className="uppercase">YouTube tutorials</h3>
+            <p>
+              Watch engineers walk through code examples and developer product
+              updates.
+            </p>
+          </a>
+        </section>
+
+        <ClientLibraries />
+
+        <a name="#developer-digest" id="developer-digest">
+          &nbsp;
+        </a>
+        <h2 id="subscribe">Developer digest</h2>
+        <section className="newsletter">
+          <div className="sign-up">
+            <p>
+              Stay up to date with the latest from Stripe’s API and developer
+              products:
```

Comment:
I think this may have been my bug - this needs a space

```suggestion
              products:{' '}
```